tests/integration/groups.spec.ts

```diff
@@ -0,0 +1,10 @@
+import {test, expect} from "../admin-pages";
+
+test.describe('Groups', async () => {
+    test('loads page', async ({ groupsPage }) => {
+        await groupsPage.goto();
+        await expect(groupsPage.getActiveNavItem).toContainText('Groups')
+        await expect(groupsPage.getGroupsList).toBeVisible();
+        await expect(groupsPage.getAddGroupForm).toBeVisible();
+    })
+});
```

Comment:
2 of 2 actual tests in this PR - load the groups page and make sure things are visible